.github/workflows/identify-codeowners.yml

```diff
@@ -0,0 +1,20 @@
+name: Identify Codeowners
+
+on:
+  pull_request_target:
+    types: [opened, synchronize, reopened, ready_for_review]
+
+jobs:
+  identify-codeowners:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Setup environment
+        uses: metamask/github-tools/.github/actions/setup-environment@1d657e262aea7e3f216754febb624831527d2565
```

Comment:
This is going to have not a **direct** merge conflict, but an **indirect** merge conflict with #29979, so let's watch out for which one merges first.

Reply:
Will do 🫡